llvm/test/CodeGen/X86/global-variable-partition-with-dap.ll

```diff
@@ -0,0 +1,42 @@
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
+
+;; A minimal test case. llc will crash if global variables already has a section
```

Comment:
Is the behavior of this test affected by this PR? If not, commit it separately as a standalone change.